CHANGELOG.md

```diff
@@ -113,6 +113,12 @@ Mandatory release that includes:
 - [#3955](https://github.com/ChainSafe/forest/pull/3955) Added support for the
   NV22 _Dragon_ network upgrade, together with the required state migration.
 
+### Changed
+
+- [#3976](https://github.com/ChainSafe/forest/pull/3976) `forest-wallet`
```

Comment:
If this is merged after https://github.com/ChainSafe/forest/pull/4152 I believe the position of this log entry needs to be manually adjusted